src/java.smartcardio/share/classes/sun/security/smartcardio/PCSCException.java

```diff
@@ -35,7 +35,7 @@
  * @since   1.6
  * @author  Andreas Sterbenz
  */
-final class PCSCException extends Exception {
+final public class PCSCException extends Exception {
```

Comment:
Use blessed modifiers order
```suggestion
public final class PCSCException extends Exception {
```

Reply:
Thank you for feedback, I have adjusted the order